tmdbsimple/base.py

```diff
@@ -27,9 +27,11 @@ class TMDB(object):
     URLS = {}
 
     def __init__(self):
-        from . import API_VERSION
+        from . import API_VERSION, REQUESTS_SESSION, __version__
         self.base_uri = 'https://api.themoviedb.org'
         self.base_uri += '/{version}'.format(version=API_VERSION)
+        self.session = REQUESTS_SESSION or requests.Session()
+        self.session.headers.setdefault('user-agent', 'tmdb_api/{}.{}.{}'.format(*__version__))
```

Comment:
Are you trying to set `user-agent` to `tmdb_api/1.4.1`, assuming the version is 1.4.1?  Here's the resulting string given the above code.
```
>>> __version__ = '1.4.1'
>>> r = 'tmdb_api/{}.{}.{}'.format(*__version__)
>>> r
'tmdb_api/1...4'
```

How about omitting `user-agent` altogether?